tests/job/test_scriptjob.py

```diff
@@ -0,0 +1,30 @@
+import os
+import unittest
+
+from pyiron_base import Project
+
+class TestScriptJob(unittest.TestCase):
```

Comment:
Similarly here. 

I don't object if anyone wants to go through and implement the new parentage everywhere it applies, but IMO it's sufficient if we just update things as we come across them.